templates/index.ts.jinja2

```diff
@@ -0,0 +1,7 @@
+{%- for version in versions -%}import * as {{ version }} from './{{ version}}';{{ "\n" }}{% endfor %}
```

Comment:
No need to put it all in one line - for readability, you can insert newline before and after `import`, just play with dashes (`{%-` vs `{%` and `-%}` vs `%}`) to make the generated file look good.